modular_skyrat/modules/cortical_borer/code/cortical_borer.dm

```diff
@@ -37,62 +37,60 @@
 	var/mob/living/simple_animal/cortical_borer/cb_inside = carbon_target.has_borer()
 	if(!cb_inside)
```

Comment:
All in all, most of this proc should be extracted in its own thing so it's not duplicated in two different files.